cmd/monaco/download/download_configs.go

```diff
@@ -240,14 +244,16 @@ func copyConfigs(dest, src project.ConfigsPerType) {
 
 // shouldDownloadClassicConfigs returns true unless onlySettings or specificSchemas but no specificAPIs are defined
 func shouldDownloadClassicConfigs(opts downloadConfigsOptions) bool {
-	return !opts.onlySettings && (len(opts.specificSchemas) == 0 || len(opts.specificAPIs) > 0)
+	return !opts.onlyAutomation && !opts.onlySettings && (len(opts.specificSchemas) == 0 || len(opts.specificAPIs) > 0)
 }
 
 // shouldDownloadSettings returns true unless onlyAPIs or specificAPIs but no specificSchemas are defined
 func shouldDownloadSettings(opts downloadConfigsOptions) bool {
-	return !opts.onlyAPIs && (len(opts.specificAPIs) == 0 || len(opts.specificSchemas) > 0)
+	return !opts.onlyAutomation && !opts.onlyAPIs && (len(opts.specificAPIs) == 0 || len(opts.specificSchemas) > 0)
 }
 
-func shouldDownloadAutomationResources() bool {
-	return featureflags.AutomationResources().Enabled()
+func shouldDownloadAutomationResources(opts downloadConfigsOptions) bool {
```

Comment:
as this PR won out, could you please add some test coverage for the new logic? The other PR included some tests - maybe they can just be transferred over :)